# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build libxext-build"

PKG_CONFIG_PATH=/x11/share/pkgconfig pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.63; am-1.11; lt-2.2.6)
libtoolize -c -f; aclocal -I m4; autoconf; autoheader; automake -a -c
PKG_CONFIG_PATH=/x11/lib/pkgconfig LDFLAGS=-Wl,-rpath,/x11/lib ./configure --disable-static --prefix=/x11

# build
make

# install
install -d $DESTDIR/x11/include/X11/extensions
install -m 644 include/X11/extensions/XInput.h $DESTDIR/x11/include/X11/extensions
install -m 644 include/X11/extensions/XInput2.h $DESTDIR/x11/include/X11/extensions
install -d $DESTDIR/x11/lib
install src/.libs/libXi.so.6.* $DESTDIR/x11/lib/libXi.so.6
ln -sf libXi.so.6 $DESTDIR/x11/lib/libXi.so
install -d $DESTDIR/x11/lib/pkgconfig
install -m 644 xi.pc $DESTDIR/x11/lib/pkgconfig

# build.pc
install -d $DESTDIR/x11/share/pkgconfig
cat > $DESTDIR/x11/share/pkgconfig/libxi-build.pc << EOF
Name: libXi
Version: 1.3
Description: X input library
Requires: $REQUIRES

devel=\\
/x11/include/X11/extensions/XInput.h \\
/x11/include/X11/extensions/XInput2.h \\
/x11/lib/libXi.so \\
/x11/lib/pkgconfig/xi.pc

exec=\\
/x11/lib/libXi.so.6
EOF

exit
--- libXi-1.3.orig/configure.ac
+++ libXi-1.3/configure.ac
@@ -3,6 +3,7 @@
 AC_PREREQ([2.57])
 
 AC_INIT(libXi, 1.3, [https://bugs.freedesktop.org/enter_bug.cgi?product=xorg], libXi)
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([dist-bzip2 foreign])
 AM_MAINTAINER_MODE
 
